### PR TITLE
Set 6 - new password

### PR DESCRIPTION
**Pull request by lespacedunmatin**
Be able to request a new password on the profile page.

### CODE REVIEW

File: src/components/PassphraseForm.jsx [hunk 1]
```diff
@@ -79,9 +89,38 @@ class PassphraseForm extends Component {
...
+        {!passphraseNewRequesting && <a
+          href='#'
+          className={styles['password-reset-link']}
+          onclick={() => this.handleForgot()}>
           {t('ProfileView.password.reset_link')}
-        </a>
+        </a>}
+
+        {passphraseNewRequesting &&
+        <span className={styles['passphrase_new_send']}>
```
Comment (GoOz) [inline src/components/PassphraseForm.jsx]: You should call for Loading component instead.
Like this https://github.com/cozy/cozy-settings/blob/master/src/components/DevicesView.jsx#L21
And then, if the default style for the Loading component isn't quite what you want, add an overwriting declaration for `.set-loading` in the `.styl` of the view.

File: src/components/PassphraseForm.jsx [hunk 2]
```diff
@@ -79,9 +89,38 @@ class PassphraseForm extends Component {
...
+          {t('ProfileView.password.reset_sending')}
```
Comment (GoOz) [inline src/components/PassphraseForm.jsx]: I guess this text won't be needed if you use Loading component.

File: src/styles/fields.styl [hunk 3]
```diff
@@ -76,3 +76,13 @@
                 background    emerald
             &::-moz-progress-bar
                 background    emerald
+
+    .passphrase_new_send
```
Comment (GoOz) [inline src/styles/fields.styl]: See my previous comment on using Loading component to adjust this.

File: src/styles/fields.styl [hunk 4]
```diff
@@ -76,3 +76,13 @@
...
+        margin-top .5em
+
+        &:before
+            @extend $icon-spinner-blue
+            @extend $icon-16
+            opacity 1
+            left -8px
+            position relative
```
Comment (GoOz) [inline src/styles/fields.styl]: I forgot this. There should be a blank line at the end.

Comment (lespacedunmatin) [inline src/styles/fields.styl]: Corrected by fbe6d84.

**Comment by GoOz** [comment]
Yeah ok, after discussing the Loading issue, I see got it all wrong, just ignore my comments regarding this.

**Comment by gregorylegarec** [comment]
Thanks @lespacedunmatin !

**Pull request merged**